esctest/tests/save_restore_cursor.py

```diff
@@ -133,7 +153,10 @@ def test_SaveRestoreCursor_Wrap(self):
     # See if we're wrapping.
     esccmd.CUP(Point(GetScreenSize().width() - 1, 1))
     escio.Write("abcd")
-    AssertEQ(GetCursorPosition().y(), 2)
+    if escargs.args.expected_terminal == "xterm":
```

Comment:
What's the reason for the special xterm behavior? It's nice to document things like this in a comment.

Reply:
That's answered already in the docstring.

Reply:
> Older versions of xterm did not save/restore the last-column flag in DECRC, causing the cursor to be the second column rather than the first…

If I understand correctly, the test as written will pass for "older versions" of xterm and (presumably) fail on the latest version. Wouldn't it make more sense to test the current code?

Reply:
It's testing the current code.  I changed this about a year ago

https://invisible-island.net/xterm/xterm.log.html#xterm_328

* modify DECRC to save/restore xterm's last-column flag used to control wrapping behavior rather than manipulating DECAWM (report/analysis by Mattias Engdegård).